[Infra UI] Metrics Explorer Documentation (#37040)
elastic/kibana: docs/infrastructure/metrics-explorer.asciidoc

[role="xpack"]
[[metrics-explorer]]

> The metrics explorer gives the user the ability to easily visualize Metricbeat data and group by arbitary attributes. It empowers the user to visualize multiple metrics and can be a jumping off point for further investigations.

Reviewer: Good additions here! We generally want to use "you" instead of "the user". What about this suggestion:
```suggestion
The metrics explorer allows you to easily visualize Metricbeat data and group it by arbitary attributes. This empowers you to visualize multiple metrics and can be a jumping off point for further investigations.
```

[role="xpack"]
[[metrics-explorer]]
== Using the Metrics explorer

> The metrics explorer allows you to create system load charts for each host you're monitoring with Metricbeat.

Reviewer: Is there anything else we can add here that explains "why" someone would use this page?

Author: I change this text to be a little more generic since the metric explorer can do more then create charts for "system load". I'm open to more suggestions, at the moment I'm having a hard time articulating the "why".

[role="screenshot"]
image::infrastructure/images/metrics-explorer-screen.png[Metrics Explorer in Kibana]

[float]
[[metrics-explorer-requirements]]

> === Metrics explorer requirements and considerations

Reviewer: Moved reqs/considerations up to the top, I think it makes more sense to outline them first
